roles/gluster_volume_create/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# tasks file for gluster_volume_create
+- name: Peer probe the nodes
+  gluster_peer:
+        state: present
+        nodes: "{{ gluster_hosts }}"
+
+- name: Create a volume
+  gluster_volume:
+        state: present
+        volume: "{{ volname }}"
```

Comment:
Same with the name of the volume. Also you may have to test what happens if arbiter_count is set to 0. Or disperse is set to 0.